spec/services/course/material/zip_download_service.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Course::Material::ZipDownloadService do
+  let(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    describe '#download' do
+      let(:folder) { create(:folder) }
+      subject { Course::Material::ZipDownloadService.download_and_zip(folder, []) }
+
+      it 'downloads the contents' do
+        pending 'To be implemented'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.